org.lflang/src/org/lflang/generator/c/CCmakeGenerator.java

```diff
@@ -225,8 +226,13 @@ CodeBuilder generateCMakeCode(
                     cMakeCode.pr("add_link_options( "+compilerFlag+")");
             }
         }
-        cMakeCode.newLine();
 
+        if (targetConfig.schedulerType == SchedulerOption.QS) {
```

Comment:
I think it's better to handle this [here](https://github.com/lf-lang/lingua-franca/blob/d561add3e18a2e65b8648f240a4c7609819eb653/org.lflang/src/org/lflang/generator/c/CGenerator.java#L864) with a 
```java
if (targetConfig.schedulerType == SchedulerOption.QS) {
    targetConfig.compileDefinitions.put("SCHEDULER_QS", "");
}
```

Reply:
Great, thanks! This is indeed a cleaner approach.